experiments/state/src/main/scala/org/apache/gearpump/streaming/state/user/api/PersistentTask.scala

```diff
@@ -64,15 +65,18 @@ abstract class PersistentTask[T](taskContext: TaskContext, conf: UserConfig)
     checkpointManager
       .recover(timestamp)
       .foreach(state.recover(timestamp, _))
+    appMaster ! CheckpointClock(taskId, timestamp)
```

Comment:
`CheckpointClock` in `onStart` is fired only once. That in`onNext` is `O(GetUpStreamMinClock)`